pykickstart/commands/autopart.py

```diff
@@ -48,6 +110,9 @@ def parse(self, args):
         return self
 
 class F9_AutoPart(FC3_AutoPart):
+    """
+        ``autopart [--encrypted [--passphrase=<phrase>]]``
```

Comment:
Does this mean that the docs for F9_AutoPart will only include the paragraphs for --encrypted and --passphrase= ?